nomenclature/processor/region.py

```diff
@@ -563,11 +573,31 @@ def _compare_and_merge(original: pd.Series, aggregated: pd.Series) -> IamDataFra
 
     # drop rows that are not in conflict
     compare = compare.dropna()
-    compare = compare[~np.isclose(compare["original"], compare["aggregated"])]
-
+    compare = compare[
+        ~np.isclose(compare["original"], compare["aggregated"], rtol=0.01)
+    ]
+    compare["relative difference (%)"] = 100 * np.abs(
```

Comment:
Please don't do number-formatting as hard-code - if at all, do that when printing the dataframe to the log.

Reply:
Do you mean leaving out the `100` to make it a percent value?

Reply:
Ignore my comment, I misunderstood the code.